nautobot_ssot/tests/device42/unit/test_utils_nautobot.py

```diff
@@ -153,13 +153,13 @@ def test_update_custom_fields_add_cf(self):
         test_site = Location.objects.create(
             name="Test", location_type=LocationType.objects.get_or_create(name="Site")[0], status=self.status_active
         )
-        self.assertEqual(len(test_site.get_custom_fields()), 3)
+        self.assertEqual(len(test_site.get_custom_fields()), 4)
         mock_cfs = {
             "Test Custom Field": {"key": "Test Custom Field", "value": None, "notes": None},
```

Comment:
Perhaps?

```suggestion
            "Test Custom Field": {"key": "Test_Custom_Field", "value": None, "notes": None},
```

Reply:
This part of the test is actually what the information from Device42 looks like and isn't the key for the CustomField like you might think. The name is admittedly a little confusing.